### PR TITLE
PHP 7.4, Symfony 5.0 and static analysis

### PR DESCRIPTION
**Pull request by tmihalicka**
Added requirements for PHP 7.4 and Symfony 5.0, added correct types to project, fixed test and added static analysis.

### CODE REVIEW

File: bin/grumphp_hooks/pre-commit [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# Run the hook command.
+# Note: this will be replaced by the real command during copy.
+#
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# Fetch the GIT diff and format it as command input:
+DIFF=$(git diff -r -p -m -M --full-index --staged | cat)
+
+# Run GrumPHP
+(cd "./" && printf "%s\n" "${DIFF}" | docker-compose run --no-deps --rm phystrix-bundle "vendor/bin/grumphp git:pre-commit --ansi --skip-success-output")
```
Comment (Rastusik) [inline bin/grumphp_hooks/pre-commit]: hooky uz riesime inak, je to na conflu

File: src/Command/BaseCommand.php [hunk 2]
```diff
@@ -1,42 +1,36 @@
...
+
 declare(strict_types=1);
 
-/**
- * @author    mskorupa
+/*
+ * @author mskorupa
  * @copyright PIXEL FEDERATION
- * @license:  Internal use only
+ * @license: Internal use only
```
Comment (Rastusik) [inline src/Command/BaseCommand.php]: tieto licencie by tu nemali asi byt :)

**Review by Rastusik — APPROVED** [review]

**Review by Rastusik — APPROVED** [review]

File: bin/grumphp_hooks/pre-commit (hunk 1 above)
Comment (tmihalicka) [reply bin/grumphp_hooks/pre-commit]: pozriem a fixnem to este.

File: docker-compose.yaml [hunk 3]
```diff
@@ -1,9 +1,9 @@
 version: '3.3'
 
 services:
-  phystrix-bundle:
-    container_name: phystrix-bundle
+  php-phystrix-bundle:
+    container_name: php-phystrix-bundle
```
Comment (Rastusik) [inline docker-compose.yaml:5]: este taka zmena, php davame uz nakoniec... ale nemusis menit

**Review by Rastusik — APPROVED** [review]

**Pull request merged**